### PR TITLE
Setup Type Checking

### PR DESCRIPTION
**Pull request by wazolab**

### CODE REVIEW

File: components/DiffMap.vue [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <script lang="ts">
 import type { PropType } from 'vue'
 import { shallowRef } from 'vue'
-import type { Geometry } from 'geojson'
+import type { Geometry } from '@turf/helpers'
```
Comment (frodrigo) [inline components/DiffMap.vue:4]: I prefer keeing the type from geojson, the solution is probably to cast from/to turf.

Comment (wazolab) [reply components/DiffMap.vue:4]: I think the problem is we are talking about the same type: Geometry but `booleanEqual` turf function expect his own ...
So we have a type error: 'Geometry is not assignable to type Geometry'...

Comment (frodrigo) [reply components/DiffMap.vue:4]:
```
import type { Geometry } from 'geojson'
import type { Geometry as TurfGeometry } from '@turf/helpers'
```
Is not a solution ?

Comment (wazolab) [reply components/DiffMap.vue:4]: yes it is but a bit too much no ? why not using turf type directly as it is provided ?

Comment (wazolab) [reply components/DiffMap.vue:4]: alright

**Pull request merged**